artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/FileLockNodeManager.java

```diff
@@ -388,4 +404,109 @@ protected FileLock lock(final int lockPosition) throws Exception {
       return null;
    }
 
+   private synchronized void startLockMonitoring() {
```

Comment:
Did you consider extending ActiveMQScheduledComponent instead?

Reply:
No, I didn't consider extending `ActiveMQScheduledComponent`. I just took what was on the original PR and cleaned it up so it could actually be merged.